src/components/ControlPanel/eventAnalytics.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import Button from 'components/Button';
+import Event from './event.js';
+import { Chart } from 'chart.js'
+import { Bar } from 'react-chartjs-2'
+
+
+
+export default class EventAnalytics extends React.Component {
```

Comment:
change this component to take in the data and labels through props instead of hard coding.

For example, you might pass in:

```jsx

const yearData = { 1: 100, 2: 50, 3: 25, 4: 10 };
const majorData= { "CS/CSE": 300, "Ling CS": 100, "Math": 25 };

<EventAnalytics yearData={yearData} majorData={majorData} />
```

And then use the passed in objects to derive the labels, data, and display them.